File: Sakila_Net/Sakila_Net/CS_SakilaLib/DAO/DAOStore.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOStore 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOStore.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClStore store)
	   {
            return DAOBD.DAOStore.SelectOne(store); 
	   }
	   public static int sqlInsert(ClStore store)
	   {
            return DAOBD.DAOStore.sqlInsert(store);
       }
	   public static int sqlDelete(ClStore store)
	   {
            return DAOBD.DAOStore.sqlDelete(store);
       }
	   public static int sqlUpdate(ClStore store)
	   {
            return DAOBD.DAOStore.sqlUpdate(store);
       }
   }
}

File: Sakila_Net/Sakila_Net/VB_SakilaLib/DAO/DAOStore.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOStore
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOStore
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOStore.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( store as ClStore) as Boolean
                        Return DAOBD.DAOStore.SelectOne(store)     
                    End Function
                    Public Shared  Function SqlInsert( store as ClStore) as Integer
                        Return DAOBD.DAOStore.SqlInsert(store)     
                    End Function
                    Public Shared  Function SqlDelete( store as ClStore) as Integer
                        Return DAOBD.DAOStore.SqlDelete(store)     
                    End Function
                    Public Shared  Function SqlUpdate( store as ClStore) as Integer
                        Return DAOBD.DAOStore.SqlUpdate(store)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
